src/guest/config_parser.cc

```diff
@@ -42,7 +42,7 @@ map<string_view, vector<string_view>> kConfigMap = {
     { kGroupAaf,     { kAafPath, kAafSuspend }},
     { kGroupPciPt,   { kPciPtDev }},
     { kGroupMed,     { kMedBattery, kMedThermal, kMedCamera } },
-    { kGroupService, { kServTimeKeep, kServPmCtrl } },
+    { kGroupService, { kServTimeKeep, kServPmCtrl, kServPowerCtrl } },
```

Comment:
better to rename with "kServVinput", it is not only for power.
Need to modify fields.md